.changeset/clear-olives-know.md

```diff
@@ -0,0 +1,177 @@
+---
+"thirdweb": minor
+---
+
+Adds a new `Bridge` module to the thirdweb SDK to access the Universal Bridge.
+
+## Features
+
+### Buy & Sell Operations
+
+The Bridge module makes it easy to buy and sell tokens across chains:
+
+- `Bridge.Buy` - For specifying the destination amount you want to receive
+- `Bridge.Sell` - For specifying the origin amount you want to send
+
+Each operation provides two functions:
+1. `quote` - Get an estimate without connecting a wallet
+2. `prepare` - Get a finalized quote with transaction data
+
+#### Buy Example
+
+```typescript
+import { Bridge, toWei } from "thirdweb";
+
+// First, get a quote to see approximately how much you'll pay
+const buyQuote = await Bridge.Buy.quote({
+  originChainId: 1, // Ethereum
+  originTokenAddress: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE", // ETH
```

Comment:
Would use the constant here. That's our constant too so would not expect ppl to know/understand it.

Another option would be to make the address optional, and only use tokenAddress for erc20.

Wdyt

Reply:
Will swap out for the constant, trying to keep the query params explicit as much as possible to reduce confusion